test-support/page-object/properties/text-array.js

```diff
@@ -0,0 +1,44 @@
+import Descriptor from '../descriptor';
+import { findElementWithAssert, trim } from '../helpers';
+
+/**
+ * Gets an array whose members are matched elements' text
+ *
+ * @param {Object} target - Component that owns the property
+ * @param {string} key - Name of the key associated to this property
+ * @param {Object} options - Additional options
+ * @param {string} options.selector - CSS selector of the element to check
+ * @param {string} options.scope - Overrides parent scope
+ * @return {array} array of text strings for the selected elements
+ */
+function getTextArray(target, key, options) {
+  let elements = findElementWithAssert(options, target);
+
+  let textStrings = new Array(elements.length);
+  for (var i=0, l=textStrings.length; i<l; i++) {
```

Comment:
have tried this (as wasn't the original author of the function and also thought it strange), but `elements` is not a real array so `map`, etc. don't work